diagnostic.md

```diff
@@ -45,22 +53,32 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+let weather; => Declares a variable weather.
+weather = 'sunny' => Assigns a value to the variable weather.
+weather === 'sunny' => checks if weather is equal to sunny.
 
 ## Question 5
 
 ```js
 //We'll learn about require later in the course
 const ask = require('../lib/ask.js');
 
-let answer = 'not empty';
+let answer = 0;
+answer = ask("Guess my secret? ");
 
-while (answer !== '') {
-  answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
-   console.log("You guessed it!");
-   break;
+if( answer < 1 && answer > 10 )
+{
+  console.log( "Guess needs to be between 1 and 10" )
+}
+else {
+  while ( answer >= 0 && answer <= 10 )
+  {
+      console.log( "You guessed it!" );
+      break;
+    }
```

Comment:
here's 1 possible solution, please read this over and make sure you understand what's happening:
```
const ask = require('../lib/ask.js');

let answer = '';

while (answer !== '5') {
  if (answer < 1 || answer > 10) {
   console.log('Please try a number between 1 and 10');
 } else if (answer !== '') {
   console.log('That\'s not it!');
 }
  answer = ask("Guess my secret? ");
}
console.log("You guessed it!");
```